launch/hardware/sensors/depth_to_laser.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+
+<launch>
+
+    <!-- Get the machine file -->
+    <arg name="machine" default="localhost"/>
+    <include file="$(env ROBOT_BRINGUP_PATH)/machines/$(arg machine).machine" />
+
+    <arg name="robot_name" />
+    <arg name="camera_info_depth" default="/$(arg robot_name)/head_rgbd_sensor/depth_registered/camera_info" />
+    <arg name="image_depth" default="/$(arg robot_name)/head_rgbd_sensor/depth_registered/image" />
+    <arg name="scan_projected" default="/$(arg robot_name)/head_rgbd_sensor/scan" />
```

Comment:
We always run this in NS in the parent launch file. So no need for this robot NS prefix.